Lib/test/test_os.py

```diff
@@ -3861,17 +3861,17 @@ def test_oserror_filename(self):
         for filenames, func, *func_args in funcs:
             for name in filenames:
                 if not isinstance(name, (str, bytes)):
-                    with self.assertRaises(TypeError):
-                        func(name, *func_args)
+                    # don't test TypeError here
```

Comment:
Other types are already well tested in other test methods of test_os.

Reply:
Then why these filenames are occurred here at first place? `bytearray` and `memoryview` should not be added in `self.bytes_filenames`.

BTW, does checking condition `sys.platform == "win32"` above still have sense?

Reply:
I removed bytearray and memoryviews from ``self.bytes_filenames`` (and ``self.filenames``).

I removed the code specific to Windows: let's see how CIs like my updated change :-)